en/flight_controller/raspberry_pi_pilotpi_rpios.md

```diff
@@ -227,15 +227,16 @@ It is also possible to just compile the code with command:
 
 You need to check these extra items to get your vehicle work properly.
 
-#### Mixer file
```

Comment:
This RaPi setup suggests you use the pilotpi_xx.conf that loads the quad main mixer and maps it to /dev/pwm_output0 as shown below.

How does this get fixed - i.e. I know that you'd assign a geometry - just not sure how it gets mapped to outputs and what you do with the configuration file.

Reply:
This is now the same as for pixhawks: you will have an output page for the rpi's pwm output driver.
You might have to set `CA_AIRFRAME` though.

Reply:
Yes you have to set `CA_AIRFRAME` as well. We'll need to improve that. It should be using the existing startup scripts and airframes.

Reply:
Thanks. Updated this bit. @bkueng Can you please check the modification. We will have to fix this up when you sort the startup scripts.